core/pom.xml

```diff
@@ -32,8 +32,8 @@
     <description/>
 
     <properties>
-        <maven.compiler.target>1.7</maven.compiler.target>
-        <maven.compiler.source>1.7</maven.compiler.source>
+        <maven.compiler.target>1.8</maven.compiler.target>
+        <maven.compiler.source>1.8</maven.compiler.source>
```

Comment:
Can we remove these 1.7 compiler limitations? If not it is not a big deal, really, but it would be awesome if we could get rid of those as there are several features, such as default methods, that have been introduced in later JDK versions and that are not available in 1.7.

Reply:
We cannot: e.g. EAP adapters have 1.7 as mandatory JDK and all the dependencies including core have to adhere to this.

Reply:
Reverted